src/struct_info.rs

```diff
@@ -574,11 +580,31 @@ impl CommonDeclarationSettings {
     }
 }
 
+/// Setting of the `into` argument.
+#[derive(Debug)]
+pub enum IntoSetting {
+    /// Setting was not set.
+    Unset,
+    /// Setting was set without a value.
+    Set,
+    /// Setting was set with a specific value (type).
+    Type(syn::ExprPath),
```

Comment:
I don't like these variant names. They represent the syntactic way this setting is configured, and I think we should use names that represent the meaning of the setting. I suggest `NoConversion`, `GenericConversion` and `ConversionToSpecificType`.

Reply:
Changed, thanks